src/develop/blend.c

```diff
@@ -1113,13 +1114,13 @@ static void _blend_screen(const _blend_buffer_desc_t *bd, const float *a, float
       float lmin = 0.0, lmax, la, lb;
```

Comment:
I suggest to replace the initialization value of lmin to float (here and in the subsequent code parts).

Reply:
This has been fixed too, now.